runner/scheduler.py

```diff
@@ -258,23 +258,29 @@ def setup_iterator(self, test_group):
             self.travel_to_dt(self.master_log.log[-1].timestamp)
```

Comment:
I don't think it matters from a functional standpoint, but maybe it will confuse the user? If the reset policy is hard, then this timestamp will probably correspond to a test that will be reset.

Reply:
I don't want to edit the log after a hard reset by removing events that did happen and contributed to tokens/money spent. So rather than a complicated time resolution process, setting it to the last event (whether kept or discarded) is simple enough.